src/date-range-picker/range-picker.tsx

```diff
@@ -0,0 +1,159 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useEffect } from 'react';
+import Calendar, { DayIndex } from './calendar';
+import InternalBox from '../box/internal';
+import SpaceBetween from '../space-between/index.js';
+
+import RelativeRangePicker from './relative-range';
+import ModeSwitcher from './mode-switcher';
+import { useDateRangePicker } from './use-date-range-picker';
+import { DateRangePickerBaseProps } from './interfaces';
+import { fireNonCancelableEvent } from '../internal/events/index.js';
+import InternalAlert from '../alert/internal';
+import LiveRegion from '../internal/components/live-region';
+import styles from './styles.css.js';
+
+export interface DateRangePickerDropdownProps
+  extends Pick<
```

Comment:
Same question as in the other PR: https://github.com/cloudscape-design/components/pull/75#discussion_r930921917

Instead of hand picking properties, it is easier to create and merge multiple base types. (Exception: it could be okay to `Pick` 1-2 props, but 11 is too much)

Reply:
Noted. Refactored the interface for this component as per the suggestion